packages/@webex/widget-provider/src/store.ts

```diff
@@ -0,0 +1,28 @@
+import {makeAutoObservable, observable} from 'mobx';
```

Comment:
each substore are logical chunks lets name them based on features 

AgentStore , taskStore etc 
 we can have a call to talk about this

Reply:
Hi @arun3528 

I agree. We would need to do that, except that we won't do it right away. At the start, we will keep it limited to just one common store and later when we build the Tasks widget, we will introduce a task store as part of that PR, agent store as part of that PR and so on, although for AgentLogin, we need not have a separate store. Just the common store shall be good to consume. 

Do we want to keep this a non-blocking comment and address it as part of future PRs?

Reply:
sounds good to me